server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java

```diff
@@ -772,6 +782,15 @@ private synchronized void replayLogs(long targetRevision) {
         }
     }
 
+    private void updateZkCommandStatusLater(UpdateServerStatusCommand command) {
+        canReplicate = command.serverStatus().replicating();
+        // Use a separate executor since executorStatusManager.updateStatus() may stop the executor that calls
+        // this method.
+        ForkJoinPool.commonPool().execute(() -> {
```

Comment:
Can't we do this only when `if (!canReplicate) {...}`?

Reply:
The `command` may update `writable` status.

Reply:
> The command may update writable status.

Isn't is ok to use the Zookeeper executor because it only sets the `writable` value to `true`? (i.e. it doesn't do anything with the Zookeeper executor.)

Reply:
I misunderstood your comment before. It is possible to use the Zookeeper executor.